### ISSUE TITLE
mega.nz plugin

### ISSUE THREAD

**Issue by styxxx** [syncany/syncany#512]
A plugin for using the MEGA storage would be nice. There is an SDK & documantion https://mega.nz/#dev

**Comment by pimotte** [comment]
Plugin requests are currently on hold, since we're focussing on the core. Feel free to give it a shot yourself. We have the infrastructure to include third-party plugins in Syncany.

**Comment by moralesalexanderj** [comment]
Yes, that will be a good plugin for all MEGA users like us. Keep doing this good work with Syncany!
